src/components/themed/WalletProgressIcon.js

```diff
@@ -73,7 +72,7 @@ export class WalletProgressIconComponent extends React.PureComponent<Props, Stat
         backgroundColor={theme.walletProgressIconBackground}
         rotation={0}
       >
-        {() => <Image style={iconSize} source={{ uri: icon }} />}
+        {() => (icon ? <Image style={iconSize} source={{ uri: icon }} /> : <View style={iconSize} />)}
```

Comment:
Why is there an in-line function here? This component is rendered many times on the Wallet list and will always re-render due to the in-line function

Reply:
The inline function is how it works for that library for it to have children. I actually did try to remove that before but could not make it work. See examples at https://github.com/bartgryszko/react-native-circular-progress. In the actual source code of it actually is a function `  children: PropTypes.func`.